### DISCUSSION TITLE
ns create -- Unable to authenticate, need: Basic realm="Administrator"

### DISCUSSION THREAD

**Discussion by Viet47** [NativeScript/NativeScript#10121] [Q&A] [answered]
Hello world, I would like to try Nativescript but I got some error. Could you help me please ? 
ns doctor
√ Getting environment information
No issues were detected.
√ Your ANDROID_HOME environment variable is set and points to correct directory.
√ Your adb from the Android SDK is correctly installed.
√ The Android SDK is installed.
√ A compatible Android SDK for compilation is found.
√ Javac is installed and is configured properly.
√ The Java Development Kit (JDK) is installed and is configured properly.
√ Local builds for iOS can be executed only on a macOS system. To build for iOS on a different operating system, you can use the NativeScript cloud infrastructure.
√ Getting NativeScript components versions information...
√ Component nativescript has 8.4.0 version and is up to date.
node version : v16.15.0
windows 10
When I start this cmd with powerShell :  ns create firt-app --ng
Unable to authenticate, need: Basic realm="Administrator"

When I start powerShell as admin. ng not found like cmd. 

has someone a idea ?  what meet realm "administrator" ?

**Comment by rigor789** [comment, marked as answer]
Looks like your NPM is not set up correctly, you may have your registry pointing to a different one than the default npm registry, you could try resetting it back with:

```bash
npm config set registry https://registry.npmjs.org/
```

**Reply by Viet47** [reply]
Thank you !
